boards/mips-malta/Makefile.include

```diff
@@ -0,0 +1 @@
+# This file must exist for RIOT to build.
```

Comment:
This file should at least add the include directory to RIOT's include path, so that one can include the "board.h".